commons/connector/altoida/upload_altoida_normal_vector.avsc

```diff
@@ -0,0 +1,12 @@
+{
+  "namespace": "org.radarcns.connector.altoida.AltoidaNormalVector",
+  "type": "record",
+  "name": "AltoidaNormalVector",
+  "doc": "Normal vector uses the x and y plane in the 2d orientation of a horizontal plane of the device",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "Timestamp in unix time acquired locally from the device."},
+    {"name": "timeReceived", "type": "double", "doc": "Timestamp in unix time received from Altoida API."},
+    {"name": "x", "type": "", "doc": "orientation on x plane."},
```

Comment:
Is the type missing or we do not know the data type?

Reply:
We don't know the data type (units) of the x and y plane.  We mentioned this in the email to the guy from altoida as well.